libraries/AP_HAL_ESP32/WiFiUdpDriver.cpp

```diff
@@ -303,8 +307,17 @@ void WiFiUdpDriver::initialize_wifi()
 
     ESP_ERROR_CHECK(esp_wifi_set_mode(WIFI_MODE_STA) );
     ESP_ERROR_CHECK(esp_wifi_set_config(WIFI_IF_STA, &wifi_config) );
+
+    // WiFi Stability Improvements
+#ifdef WIFI_POWER_SAVE_NONE
```

Comment:
If this is a new define then treat it similarly to the IP address one.  Do not use `ifdef` wherever you can use `if` and a `ifndef` to set a default value.